src/datasets/load.py

```diff
@@ -1621,6 +1621,7 @@ def dataset_module_factory(
     data_dir: Optional[str] = None,
     data_files: Optional[Union[Dict, List, str, DataFilesDict]] = None,
     trust_remote_code: Optional[bool] = None,
+    _ignore_default_config=False,
```

Comment:
Why the leading underscore?

Reply:
I would suggest naming it differently, to make clear that config_name was passed or not: if config_name is passed, we don't even care whether DEFAULT_CONFIG_NAME is set or not.

Reply:
The underscore is to signify this is an internal parameter (not supposed to be used by users)

Reply:
I think I like this name: if `_require_default_config_name=True` we require the builder class to have a default config name. If the user doesn't specify a config name we set it to `True`

Reply:
> The underscore is to signify this is an internal parameter (not supposed to be used by users)

I mean, it is the first time I see a "private" function parameter... Wondering if that makes sense...

Reply:
It's not a good practice :/

I hesitated to not have the feature to use the parquet export if the user does specify a config name explicitly just because I don't like doing "private" arguments

Reply:
I see...